scripts/setup_github_app.py

```
@@ -0,0 +1,240 @@
+    print(f"\n{number}. {text}")
+    print("-" * 40)
+
+
+def check_env_file():
+    """Check if .env file exists and create from example if not."""
+    env_path = Path(".env")
+    env_example_path = Path(".env.example")
+
+    if not env_path.exists():
+        if env_example_path.exists():
+            print("Creating .env file from .env.example...")
```

Ruff (0.11.9)

`print` found

Remove `print`

(T201)